setup.py

```diff
@@ -90,10 +90,13 @@ def build_extension(self, ext):
 
         elif platform.system() == "Windows":
             cmake_args += [
-                '-G Visual Studio 17 2022',
+                '-G Visual Studio 17 2022',"-DCMAKE_CXX_STANDARD=20"
```

Comment:
This should be part of the CMake file as a requirement for building Triton, why is it set here?

Reply:
> This should be part of the CMake file as a requirement for building Triton, why is it set here?

Please look at where you have any questions.

Reply:
> This should be part of the CMake file as a requirement for building Triton, why is it set here?

The reason I put it here is that when I'm testing it, to be consistent with the llvm compiled version, this can be passed to the cmake configuration. Fixed.

Reply:
> This should be part of the CMake file as a requirement for building Triton, why is it set here?

Assume that if you are using c++ 17compile to llvm ,compile triton project also needs to c++ 17, example:using c++ 11 or c++ 20 other versions may be wrong